src/breadcrumb-group/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { RenderResult } from '@dojo/framework/core/interfaces';
+import theme from '@dojo/framework/core/middleware/theme';
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+import { Breadcrumb, BreadcrumbItem } from '../breadcrumb';
+import * as css from '../theme/default/breadcrumb-group.m.css';
+import * as fixedCss from './styles/breadcrumb-group.m.css';
+
+export interface BreadcrumbGroupProperties {
+	items: BreadcrumbItem[];
+	label: string;
+}
+
+export interface BreadcrumbGroupChildren {
+	(items: BreadcrumbItem[]): RenderResult;
+}
+
+const factory = create({ theme })
+	.properties<BreadcrumbGroupProperties>()
+	.children<BreadcrumbGroupChildren | undefined>();
+
+export const BreadcrumbGroup = factory(function BreadcrumbGroup({
+	children,
+	properties,
+	middleware: { theme }
+}) {
+	const { items, label } = properties();
+	const themeCss = theme.classes(css);
+
+	const defaultRenderer: BreadcrumbGroupChildren = (items: BreadcrumbItem[]) => {
+		const lastIndex = items.length - 1;
+
+		return (
+			<ol classes={[fixedCss.listFixed, themeCss.list]}>
```

Comment:
i think this would work a lot better if the `ol` was rendered outside of the renderer and the outer nodes of the `Breadcrumb` widgets were `li`. Then users would always benefit from the ordered list semantics and the custom renderers etc would look a lot cleaner / simpler.

Reply:
Ok. I hadn't done it that way because it requires the user to know the internals of `<BreadcrumbGroup` and `<Breadcrumb>` to ensure they use them correctly to end up with semantically-correct markup, but I can add a note to the README to that effect.

Reply:
The reason I did not do it that way is because it requires the user to know the internals of both widgets in order to end up with semantically-correct markup, and it effectively makes `<Breadcrumb>` useless since it would only be used by `<BreadcrumbGroup>`. If a user needs a custom renderer, then chances are they want control over what is rendered inside the `<li>` and would therefore need to roll their own breadcrumb item instead of using `<Breadcrumb>`.

Reply:
I get what you're saying @mwistrand , but firstly, I don't think `Breadcrumb` will be used outside of this widget and secondly, without moving the `ol` / `li` inside their widgets we won't be able to handle adding any of the fixed css styling or aria tags for them which is an important part of the widget in my opinion. We can provide them with a `BreadCrumbSeperator` widget too if that helps which simply applies `aria` and returns an `li` with the appropriate contents. This would give us the most flexible, user friendly approach.